### PR TITLE
Clarify usage of python -m particle {search,pdgid}

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Addresses issue https://github.com/scikit-hep/particle/issues/91 - basically a typo in the documentation.

I now added some examples to the help printout.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -206,7 +206,7 @@ You can quickly search for particles from the command line with:
 
 .. code-block:: bash
 
-    $ python -m particle search 'K*0'
+    $ python -m particle search K*0
```
Comment (henryiii) [inline README.rst]: With the changed version, won't a shell try to expand `K*0` into `Kfile0` if you have a `Kfile0` file in your directory?

Comment (eduardo-rodrigues) [reply README.rst]: I've just retried on my Windows laptop and on SWAN (linux).
Windows:
```
(base) C:\Users\Eduardo Rodrigues>python -m particle search K*0
<Particle: name='K*(892)0', pdgid=313, mass=895.55 ± 0.20 MeV>
<Particle: name='K*(1680)0', pdgid=30313, mass=1718 ± 18 MeV>
<Particle: name='K*(1410)0', pdgid=100313, mass=1421 ± 9 MeV>
(base) C:\Users\Eduardo Rodrigues>python -m particle search 'K*0'
Particle 'K*0' not found.
```
Linux on SWAN:
```
bash-4.2$ python -m particle search K*0
<Particle: name='K*(892)0', pdgid=313, mass=895.55 ± 0.20 MeV>
<Particle: name='K*(1680)0', pdgid=30313, mass=1718 ± 18 MeV>
<Particle: name='K*(1410)0', pdgid=100313, mass=1421 ± 9 MeV>
bash-4.2$ python -m particle search 'K*0'
<Particle: name='K*(892)0', pdgid=313, mass=895.55 ± 0.20 MeV>
<Particle: name='K*(1680)0', pdgid=30313, mass=1718 ± 18 MeV>
<Particle: name='K*(1410)0', pdgid=100313, mass=1421 ± 9 MeV>
```
So there's a difference in behaviour here. And I see no shell expansion as you feared.

I'm now tempted to add a warning message to the message, to be clear.

Comment (eduardo-rodrigues) [reply README.rst]: Just to make it funnier, the following does work on Windows:
```
(base) C:\Users\Eduardo Rodrigues>python -m particle search "K*0"
<Particle: name='K*(892)0', pdgid=313, mass=895.55 ± 0.20 MeV>
<Particle: name='K*(1680)0', pdgid=30313, mass=1718 ± 18 MeV>
<Particle: name='K*(1410)0', pdgid=100313, mass=1421 ± 9 MeV>
```
I did not know Windows treats single and double quotes differently, meaning I had never come across this. But that's the case.

Bottom line: a little warning is worth adding to the doc.

File: README.rst [hunk 2]
```diff
@@ -202,11 +202,12 @@ with easily recognisable names. For example:
...
     0.5
 
-You can quickly search for particles from the command line with:
+You can quickly search for particles from the command line with
+(note: quotes may be used/needed but only double quotes work as expected on Windows):
 
 .. code-block:: bash
 
-    $ python -m particle search 'K*0'
+    $ python -m particle search K*0
```
Comment (henryiii) [inline README.rst]: Let's use double quotes here. The only general danger with double quotes is that shell variables are expanded, but no particles have `$` in their name so this should be fine.

**Pull request merged**